var/spack/repos/builtin/packages/zig/package.py

```diff
@@ -17,6 +17,6 @@ class Zig(CMakePackage):
         default='Release', description='CMake build type'
     )
 
-    depends_on('llvm@11.0.0: +all_targets')
+    depends_on('llvm@11.0.0:')
```

Comment:
I would guess this also needs to be `targets=all`?

Reply:
Hm, not a Zig user, but I would assume most people would just want to build for native only, otherwise they can always to zig ^llvm targets=all?

Reply:
I'm not one either, but potentially there's code in there that assumes the upstream LLVM has been build with all targets? Pinging maintainer @alalazo but the easy solution would just to be to add `targets=all` : it has the same meaning as the previous requirement and since the new default is `targets=all` it won't change the llvm concretization.

Reply:
Zig is a bit of a special case in this respect, one of its core functions is being able to target any platform from any platform, [news piece here with some info](https://zig.news/kristoff/cross-compile-a-c-c-project-with-zig-3599).  I'd guess it may actually cause runtime failures, or at least a jarring experience, if that went away.

```suggestion
    depends_on('llvm@11.0.0: targets=all')
```